api/TextDecoder.json

```
@@ -150,6 +156,10 @@
             "chrome_android": {
               "version_added": "38"
             },
+            "deno": {
+              "version_added": "1.0",
+              "notes": "<code>stream</code> option not supported before Deno 1.11."
```

The technically correct way to handle this is with a subfeature of `decode` documenting for `options.stream`. See https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#parameters-and-parameter-object-features

But since that would require touching some non-Deno data, I'm willing to overlook it here. 😃 I'd settle for a style change:

```suggestion
              "notes": "Before Deno 1.11, <code>option.stream</code> is not supported."
```

Better still would be a little more information on what "not supported" means here. Is the option ignored? Does it throw an error?

---

Have clarified what "not supported" means and have updated the style. I can revisit this with the "proper" fix in a follow-up PR.